operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -38,6 +38,10 @@ type ImageContentSourcePolicySpec struct {
 	// (e.g. `a, b` vs. `b, a`) the configuration is not rejected but the resulting order is unspecified.
 	// +optional
 	RepositoryDigestMirrors []RepositoryDigestMirrors `json:"repositoryDigestMirrors"`
```

Comment:
Can we add a `Deprecated: use repositoryMirrors` note or some such here to [mark this as deprecated][1]?

[1]: https://github.com/golang/go/wiki/Deprecated

Reply:
Fixed. Thanks.

Reply:
You are strengthening the schema. You cannot do that. They might be objects out there in customer clusters which have duplicated sources.